volctl/osd.py

```diff
@@ -83,7 +84,18 @@ def update_values(self, volume, mute):
             self._volctl.settings.get_int("osd-timeout"), self._cb_hide_timeout
         )
 
-    def _move_to_corner(self):
+    def _move_to_position(self, position):
+        if position == "center":
+            self._move_to_center()
+        elif position == "bottom-right":
+            self._move_to_bottom_right()
+        elif position == "top-right":
+            self._move_to_top_right()
+        else:
+            print("Not supported osd-position. Supported are: center, bottom-right, top-right; osd-position=" + position)
+            raise ValueError
+
+    def _move_to_bottom_right(self):
```

Comment:
`_move_to_position` should be as concise and DRY as possible.

Is there any reason you don't use the `_compute_position` method? Make sure your code has proper multi screen support!